internal/docker/build.go

```diff
@@ -61,31 +67,6 @@ func Build(
 	defer func() {
 		retErr = errors.Join(retErr, f.Close())
 	}()
-	buildStages, err := ParseDockerfileBuildStages(f)
```

Comment:
Now that we cache all the layers using buildx cache with `mode=max`, we don't need to do this any more.